Docs: Add missing Connection field (#127466)
grafana/grafana: docs/sources/shared/visualizations/datalink-options-2.md

| -------------------- | ----------------------------------------------------------------- |
| Title                | A human-readable label for the action that's displayed in the UI. |
| Confirmation message | A descriptive prompt to confirm or cancel the action. |

> | Connection           | Specify the configured data source through which to route the action's HTTP request. |

Author: ```suggestion
| Connection           | Specify how the action's HTTP request is sent. Choose from: Direct from browser or routed through a configured data source. |
```

> | Connection           | Specify the URL or configured data source through which to route the action's HTTP request. |

Reviewer: You can't actually specify the url..

<img width="503" height="358" alt="Image" src="https://github.com/user-attachments/assets/ce31ad21-b980-45f2-a2a7-ea4f0965a793" />

> | Connection           | Specify the URL or configured data source through which to route the action's HTTP request. |

Author: ```suggestion
| Connection           | Specify the configured data source through which to route the action's HTTP request. |
```

> | Connection           | Specify how the action's HTTP request is sent. Choose from: Direct from browser or routed through a configured data source. |

Reviewer: ```suggestion
| Connection           | Specify how the action's HTTP request is sent. Choose from: **Direct from browser** or routed through a configured data source. |
```
